docs/projects/ad9434_fmc/index.rst

```diff
@@ -0,0 +1,225 @@
+Instance             Zynq/Microblaze
+==================== ===============
+axi_ad9434_dma       0x44A0_0000    
+axi_ad9434_dma       0x44A3_0000    
+==================== ===============
+
+SPI connections
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+.. list-table::
+   :widths: 25 25 25 25
+   :header-rows: 1
+
+   * - SPI type
+     - SPI manager instance
+     - SPI subordinate
+     - CS
+   * - PS
+     - SPI 0
+     - AD9517
+     - 1
+   * - PS
+     - SPI 0
+     - AD9434BCPZ
+     - 0
+
+Interrupts
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Below are the Programmable Logic interrupts used in this project.
+
+================ === ========== ===========
+Instance name    HDL Linux Zynq Actual Zynq
+================ === ========== ===========
+axi_ad9783_dma   13  57         89             
+================ === ========== ===========
+
+These are the board-specific interrupts 
+(found in :git-hdl:`here <projects/ad9434_fmc/common/ad9434_bd.tcl>`).
+
+Building the HDL project
+-------------------------------------------------------------------------------
+
+The design is built upon ADI's generic HDL reference design framework.
+ADI does not distribute the bit/elf files of these projects so they
+must be built from the sources available :git-hdl:`here </>`. To get
+the source you must
+`clone <https://git-scm.com/book/en/v2/Git-Basics-Getting-a-Git-Repository>`__
+the HDL repository.
+
+Then go to the project location (**projects/ad9434_fmc/carrier**) and run the 
+make command by typing in your command prompt (this example is for 
+:xilinx:`ZC706`):
+
+**Linux/Cygwin/WSL**
+
+.. code-block::
+
+   user@analog:~$ cd hdl/projects/ad9434_fmc/zc706
+   user@analog:~/hdl/projects/ad9434_fmc/zc706$ make
+
+A more comprehensive build guide can be found in the :ref:`build_hdl` user 
+guide.
+
+Resources
+-------------------------------------------------------------------------------
+
+Systems related
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Here you can find the quick start guides available for these evaluation boards:
+
+.. list-table::
+   :widths: 20 10 20 20 20 10
+   :header-rows: 1
+
+   * - Evaluation board
+     - Zynq-7000
+     - Zynq UltraScale+ MP
+     - Microblaze
+     - Versal
+     - Arria 10
+   * - AD9434-FMC
+     - :dokuwiki:`ZC706 <resources/fpga/xilinx/fmc/ad9434>`
+     - ---
+     - ---
+     - ---
+     - ---
+
+Hardware related
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+- Product datasheets:
+
+   - :adi:`AD9434`
+   - :adi:`EVAL-AD9434`
+
+HDL related
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+-  :git-hdl:`AD9434-FMC HDL project source code <projects/ad9434_fmc>`
+
+.. list-table::
+   :widths: 30 35 35
+   :header-rows: 1
+
+   * - IP name
+     - Source code link
+     - Documentation link
+   * - axi_ad9434
```

Comment:
Please use uppercase letters for all the IP names in this list, like in the template: https://github.com/analogdevicesinc/hdl/blob/main/docs/projects/template/index.rst?plain=1#L668